### PR TITLE
Arreglar traduciones de capa de telegestion de GIS

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

> Arreglar documetnacion de telegestion en GIS

### Tasks:

- [x] Fix translations
- [x] Fix persona

### Links:

> Add links for each file modified

* [CA](http://builds.gisce.net/powerp-docs/powerp_fix_tg_gis/ca/gis/webgis/#capas-de-telegestion)
* [ES](http://builds.gisce.net/powerp-docs/powerp_fix_tg_gis/es/gis/webgis/#capas-de-telegestion)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_fix_tg_gis/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_fix_tg_gis/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/gis/webgis.md [hunk 1]
```diff
@@ -39,7 +39,7 @@
     - Aquestes capes son les que podem sobreposar a la nostra capa base i en poem selecionar múltiples a l'hora.
 #### Capes de telegestio
 
-  Aquest grup de capes ens permet veure informacio sobre la telelgesio.
+  Aquest grup de capes permet veure informacio sobre la telelgesio.
```
Comment (jaumef) [inline docs/gis/webgis.md]: No es tradueix

File: docs/gis/webgis.md [hunk 2]
```diff
@@ -39,7 +39,7 @@
...
 
   - Amb CCH: Apareixen els comptadors amb el camp “TG Contador” actiu ,  “Conectado” actiu i “Póliza Telegestionada” “Operativa con CCH”
```
Comment (jaumef) [inline docs/gis/webgis.md]: En la traducció, hi ha un "activo i poliza", hauria de ser amb "y"

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_fix_tg_gis/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_fix_tg_gis/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**